ansible/roles/beetbox-drupal/tasks/build-makefile.yml

```diff
@@ -1,6 +1,24 @@
 ---
 - name: Generate Drupal site with drush makefile.
   command: >
-    {{ drush_path }} make -y {{ drupal_makefile_path }} --no-gitinfofile
+    {{ drush_path }} make -y {{ drupal_makefile_path }} --no-gitinfofile {% if drupal_distro %}--no-recursion{% endif %}
     chdir={{ beet_root }}
   when: not drupal_site.stat.exists
+
+- name: Remove built distribution profile directory.
+  command: >
```

Comment:
I'd recommend using the file module with `state: absent` rather than command module here.